spec/support/acts_as_tenant.rb

```diff
@@ -9,6 +9,7 @@ def with_tenant(tenant, &proc)
     context "with tenant #{tenant.inspect}" do |*params|
       before(:each) do
         ActsAsTenant.current_tenant = send(tenant)
+        Capybara.app_host = "http://" + send(tenant).host
```

Comment:
Hmm OK, I'll do it in a new PR.

On Mon, Jan 5, 2015 at 5:27 PM, Joel Low <notifications@github.com> wrote:

> In spec/support/acts_as_tenant.rb
> <https://github.com/Coursemology/coursemology2/pull/25#discussion-diff-22452286>
> :
>
> > @@ -9,6 +9,7 @@ def with_tenant(tenant, &proc)
> >      context "with tenant #{tenant.inspect}" do |*params|
> >        before(:each) do
> >          ActsAsTenant.current_tenant = send(tenant)
> > +        Capybara.app_host = "http://" + send(tenant).host
>
> You might want to split the capybara config as a separate pr
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/25/files#r22452286>.
>